extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -105,7 +106,18 @@ class AmpStickyAd extends AMP.BaseElement {
         toggle(this.element, true);
         this.viewport_.addToFixedLayer(this.element);
         this.scheduleLayout(this.ad_);
+        // Add border-bottom to the body to compensate space that was taken
+        // by sticky ad, so no content would be blocked by sticky ad unit.
+        const borderBottom = this.element./*OK*/offsetHeight;
+        this.viewport_.setBorderBottom(borderBottom);
+        // TODO(zhouyx): need to delete borderBottom when sticky ad is dismissed
         this.removeOnScrollListener_();
+          timer.delay(() => {
```

Comment:
Not sure about the delay here. But I do want to introduce a delay before setting the sticky-ad background color to minimize the gap between the display of sticky-ad container and real ad inside.